src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double RESIZE = 1.5;
+    private Object[] array;
+    private int size;
+
+    public ArrayList() {
+        this.array = new Object[DEFAULT_SIZE];
+    }
+
+    public void modifiedCapacity() {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private void growIfArrayFull() {
```